components/python/python37/python-37.p5m

```diff
@@ -183,7 +180,9 @@ link path=usr/lib/$(MACH64)/libpython3.7m.so target=libpython3.7m.so.1.0
 file path=usr/lib/$(MACH64)/libpython3.7m.so.1.0
 link path=usr/lib/$(MACH64)/libpython3.7m_db.so target=libpython3.7m_db.so.1.0
 file path=usr/lib/$(MACH64)/libpython3.7m_db.so.1.0
-file path=usr/lib/$(MACH64)/libpython3.so
+# Mediate libpython3 stable ABI
+file usr/lib/$(MACH64)/libpython3.so path=usr/lib/$(MACH64)/libpython37.so
```

Comment:
bonus point for adding a comment :)

Reply:
Thank you :) Can I merge?

Reply:
Yes, the changes look good. On my build system the tests are still running. Alas I have a lot more failing tests than in the results-64.master file listed. Even a core dump on ./python -m test.regrtest --pgo